src/control_devices/CControlSurfaceBoarderBuilder.h

```diff
@@ -47,6 +47,7 @@ class CControlSurfaceBoarderBuilder
      * @param xsiLower Lower skin point, where xsi=1 is the flap leading edge, and xsi=0 the trailing edge
      */
     TopoDS_Wire boarderWithLEShape(double rLEHeight, double xsiNose, double xsiUpper, double xsiLower);
+    TopoDS_Wire boarderWithInnerShape(double rTEHeight, double xsiTail, double xsiTEUpper, double xsiTELower);
```

Comment:
Here we used the same typo ('boarder' instead of 'border') to be consistent with the surrounding code. This should probably be fixed in a separate commit.